theano/sandbox/gpuarray/multinomial.py

```diff
@@ -0,0 +1,256 @@
+# TODO test dtype != float32
+import os
+
+import pygpu
+
+import theano
+import theano.sandbox.multinomial
+from theano import Apply, config
+from theano.gof import Op, local_optimizer
+from theano.tensor import NotScalarConstantError, get_scalar_constant_value
+from theano.sandbox import gpuarray
+from .basic_ops import as_gpuarray_variable, infer_context_name
+from .fp16_help import write_w
```

Comment:
But you don't actually use it anywhere.  I'm not against making float16 support in a follow-up pass, but don't import unused stuff.